SPECS/containerized-data-importer/containerized-data-importer.spec

```diff
@@ -32,6 +32,22 @@ BuildRequires:  pkgconfig
 BuildRequires:  rsync
 BuildRequires:  sed
 ExclusiveArch:  x86_64 aarch64
+Requires:       cpio
+Requires:       curl
+Requires:       diffutils
+Requires:       findutils
+Requires:       jq
+Requires:       libnbd
+Requires:       nbdkit
+Requires:       patch
+Requires:       rsync
+Requires:       librsync
+Requires:       qemu-tools
+Requires:       qemu-img
+Requires:       shadow-utils
+Requires:       tar
```

Comment:
There's no `%files` section for the base package, which means we don't actually produce an RPM for the `containerized-data-importer` package.  [You can confirm this by looking at PMC](https://packages.microsoft.com/cbl-mariner/2.0/prod/base/x86_64/).

These requirements, then, are being attached to a non-existent package. Effectively, there's no change here. These requirements should go under the subpackages that actually require them.

Reply:
Let me check on this and revert.